src/api/app/assets/javascripts/webui/request.js

```diff
@@ -134,11 +134,10 @@ $(document).ready(function(){
 
 function loadDiffs(element){
   $('.loading-diff').removeClass('invisible');
-  var id = element.data('action-id');
-  var number = element.data('req-number');
+  var url = element.data('path');
```

Comment:
So it's a path or a URL? :smile:

I would choose one name, either `url` or `path` and stick to it.

Reply:
Haha, it's the complete URL. I'll update it. :laughing: